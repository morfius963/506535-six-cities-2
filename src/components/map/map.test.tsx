import * as React from "react";
import * as renderer from "react-test-renderer";
import Map from "./map";
import fixtureData from "../../__fixtures__/offers";

describe(`snapshot test`, () => {
  it(`Component correctly renders`, () => {
    const activeOffers = fixtureData.filter((offer) => offer.city.name === `Amsterdam`);
    const props = {
      offers: activeOffers,
      activeCard: -1,
      isInOfferDetails: false
    };
    const div = document.createElement(`div`);

    div.id = `map`;
    document.body.appendChild(div);

    const component = renderer
      .create(<Map {...props} />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
