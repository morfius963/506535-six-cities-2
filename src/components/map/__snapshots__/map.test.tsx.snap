// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test Component correctly renders 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
